src/components/breadcrumbs/styled.ts

```diff
@@ -1,36 +1,35 @@
 import { Link as RRDLink } from 'react-router-dom';
 import styled from 'styled-components';
+import { Colour, theme, Unit } from '@fellesdatakatalog/theme';
 
 const BreadcrumbsNav = styled.nav`
-  color: #335380;
+  color: ${theme.colour(Colour.BLUE, 'B60')};
   display: flex;
   flex-direction: column;
   width: 100%;
   max-width: 1140px;
   margin: 0 auto;
-  padding: 16px 16px 0 16px;
+  padding: ${theme.spacing('S16', Unit.PX)} ${theme.spacing('S16', Unit.PX)} 0 ${theme.spacing('S16', Unit.PX)};
```

Comment:
Vi foretrekker em/rem fremfor piksler og bruke 16px som baseline. Dette skalerer bedre.
https://uxdesign.cc/why-designers-should-move-from-px-to-rem-and-how-to-do-that-in-figma-c0ea23e07a15

Reply:
![image](https://user-images.githubusercontent.com/50879748/197617513-4cbe9d87-0290-43d8-89e8-9c2bb9ecf56c.jpeg)